DataFormats/EgammaReco/interface/SuperClusterSoA.h

```diff
@@ -0,0 +1,22 @@
+#ifndef EgammaReco_SuperClusterSoA_h
+#define EgammaReco_SuperClusterSoA_h
```

Comment:
Include guards should follow the same pattern as in earlier files
```suggestion
#ifndef DataFormats_EgammaReco_interface_SuperClusterSoA_h
#define DataFormats_EgammaReco_interface_SuperClusterSoA_h
```